src/components/timeline/TrackContextMenu.js

```diff
@@ -5,7 +5,7 @@
 // @flow
 import React, { PureComponent } from 'react';
 import { ContextMenu, MenuItem } from 'react-contextmenu';
-import './TrackContextMenu.css';
+import 'firefox-profiler/components/timeline/TrackContextMenu.css';
```

Comment:
For example, this import should stay like `./TrackContextMenu.css` instead of converting it to absolute path. There are others in the PR as well.